src/sparseml/modifiers/quantization/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
looking at quantization overall  it looks like a clean port and lifecycle makes sense - won't dig in further for now as long as we can get tests ported maybe working with me and @rahul-tuli